models/test_data.go

```diff
@@ -0,0 +1,109 @@
+package models
+
+var contacts = ContactDetails{
+	Email:     "test@test.co.uk",
+	Name:      "john test",
+	Telephone: "01654 765432",
+}
+
+var methodology = GeneralDetails{
+	Description: "some methodology description",
+	HRef:        "http://localhost:22000//datasets/methodologies",
+	Title:       "some methodology title",
+}
+
+var publications = GeneralDetails{
+	Description: "some publication description",
+	HRef:        "http://localhost:22000//datasets/publications",
+	Title:       "some publication title",
+}
+
+var publisher = Publisher{
+	Name: "The office of national statistics",
+	Type: "government",
+	HRef: "https://www.ons.gov.uk/",
+}
+
+var qmi = GeneralDetails{
+	Description: "some qmi description",
+	HRef:        "http://localhost:22000//datasets/123/qmi",
+	Title:       "Quality and Methodology Information",
+}
+
+var relatedDatasets = GeneralDetails{
+	HRef:  "http://localhost:22000//datasets/124",
+	Title: "Census Age",
+}
+
+var inputDataset = &Dataset{
```

Comment:
all above vars are structs, but you switch to using a pointer to a struct here - this feels inconsistent...